### PR TITLE
Ciac 10972 zoom add user sso token revoke

### PR DESCRIPTION
**Pull request by ilaredo**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-10972](https://jira-dc.paloaltonetworks.com/browse/CIAC-10972)

## Description
Added new Zoom command to support DELETE /users/\{userId}/token

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Zoom/Integrations/Zoom/README.md [hunk 1]
```diff
@@ -1919,6 +1919,23 @@ Searches chat messages or shared files between a user and an individual contact
 >| 2023-05-22T08:24:14Z | None | a62636c8-b6c1-4135-9352-88ac61eafc31 | <example@example.com> | message | admin zoom | None | uJiZN-O7Rp6Jp_995FpZGg |
 >| 2023-05-22T08:20:22Z | None | 4a59df4a-9668-46bd-bff2-3e1f3462ecc3 | <example@example.com> | my message | admin zoom | None | uJiZN-O7Rp6Jp_995FpZGg |
 
+### zoom-delete-user-token
+
+***
+Revoke a user's Zoom SSO session.
+To list all available users use `zoom-list-users` command.
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/Zoom/README.md:1926]:
```suggestion
To list all available users use the `zoom-list-users` command.
```

File: Packs/Zoom/Integrations/Zoom/Zoom.yml [hunk 2]
```diff
@@ -1225,6 +1225,15 @@ script:
       name: channel_id
     description: Delete a mirrored Zoom channel.
     name: close-channel
+  - name: zoom-delete-user-token
+    description: Revoke a user's Zoom SSO session if their account becomes compromised. Use the command zoom-list-users to get the user ID or user email.
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/Integrations/Zoom/Zoom.yml:1229]:
```suggestion
    description: Revoke a user's Zoom SSO session if their account becomes compromised. Use the zoom-list-users command to get the user ID or user email.
```

File: Packs/Zoom/ReleaseNotes/1_6_14.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Zoom
+
+- Added command:
+  - ***zoom-delete-user-token***
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/ReleaseNotes/1_6_14.md:7]:
```suggestion
Added the following command:
- ***zoom-delete-user-token***
```

**Comment by ShirleyDenkberg** [comment]
@ilaredo Doc review completed.

File: Packs/Zoom/Integrations/Zoom/Zoom.yml [hunk 4]
```diff
@@ -1225,6 +1225,15 @@ script:
       name: channel_id
     description: Delete a mirrored Zoom channel.
     name: close-channel
+  - name: zoom-delete-user-token
+    description: Revoke a user's Zoom SSO session if their account becomes compromised. Use the zoom-list-users command to get the user ID or user email.
```
Comment (eyalpalo) [inline Packs/Zoom/Integrations/Zoom/Zoom.yml:1229]:
```suggestion
    description: Revoke a user's Zoom SSO session. Use the zoom-list-users command to get the user ID or user email.
```

lets stick to the actual behavior of the command and not to the use case

File: Packs/Zoom/Integrations/Zoom/README.md [hunk 5]
```diff
@@ -1919,6 +1919,23 @@ Searches chat messages or shared files between a user and an individual contact
...
+#### Base Command
+
+`zoom-delete-user-token`
+
+#### Input
+
+| **Argument Name** | **Description**                                                                         | **Required** |
+|-------------------|-----------------------------------------------------------------------------------------| --- |
+| user_id           | The user ID or email to be revoked.                                                     | Required | 
+
```
Comment (eyalpalo) [inline Packs/Zoom/Integrations/Zoom/README.md:1938]: Lets add a command run example

**Review by eyalpalo — CHANGES_REQUESTED** [review]
Nice!

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**